src/main/java/macrobase/analysis/stats/mixture/FiniteGMM.java

```diff
@@ -38,6 +38,10 @@ public void train(List<Datum> data) {
         VariationalInference.trainMeanField(this, data, mixingComponents, clusters);
     }
 
+    /**
+     * @param datum
+     * @return log probability density of the given datum
+     */
     @Override
```

Comment:
Is the probability density of the given datum corresponds to the likelihood of the datum? Say  the probability density of the given datum is 0.9, then does it mean that 90% this datum is not an outlier.